static/app/views/dashboards/widgetBuilder/components/widgetPreview.tsx

```diff
@@ -0,0 +1,87 @@
+import PanelAlert from 'sentry/components/panels/panelAlert';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import useRouter from 'sentry/utils/useRouter';
+import {
+  type DashboardDetails,
+  type DashboardFilters,
+  DisplayType,
+} from 'sentry/views/dashboards/types';
+import {useWidgetBuilderContext} from 'sentry/views/dashboards/widgetBuilder/contexts/widgetBuilderContext';
+import {convertBuilderStateToWidget} from 'sentry/views/dashboards/widgetBuilder/utils/convertBuilderStateToWidget';
+import WidgetCard from 'sentry/views/dashboards/widgetCard';
+import WidgetLegendNameEncoderDecoder from 'sentry/views/dashboards/widgetLegendNameEncoderDecoder';
+import WidgetLegendSelectionState from 'sentry/views/dashboards/widgetLegendSelectionState';
+
+interface WidgetPreviewProps {
+  dashboard: DashboardDetails;
+  dashboardFilters: DashboardFilters;
+}
+
+function WidgetPreview({dashboard, dashboardFilters}: WidgetPreviewProps) {
```

Comment:
I didn't test this component as much because there are still so many behaviours that are still unfilled.. figured we'd test it more once we have more functional challenges.